docs/spec/wire.md

```diff
@@ -170,13 +170,16 @@ Mozilla/5.0 AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/5
 
 _Note, this is more restrictive than the User-Agent definition in [RFC 7231](https://tools.ietf.org/html/rfc7231#section-5.5.3). Requests from some browsers may not comply with these requirements as it is impossible to override browser User-Agent headers._
 
-#### 2.4.4. Header Authorization
+#### 2.4.4. Attribution Header
```

Comment:
Alternatively, did you consider using the existing user-agent mechanism to convey attribution?

Reply:
I considered it and thought that they could have slightly different meanings, e.g.:

```
User-Agent: foo/1.12.0
X-Conjure-Attribution: Project foo <foo-mailing-list@bar.com>
```

or

```
User-Agent: python-requests/2.2.1
X-Conjure-Attribution: 84273d73-7527-45ac-ac02-c2e61cbdc5b3
```

I don't feel strongly attached to having a new header.